.github/workflows/deploy-prod-static.yml

```diff
@@ -40,13 +52,26 @@ jobs:
           cp -r packages/typescriptlang-org/public site
         env:
           YARN_CHECKSUM_BEHAVIOR: ignore
+      
+      - name: Setup Pages
+          uses: actions/configure-pages@983d7736d9b0ae728b81ab479565c72886d7745b # v5
+          if: github.event_name != 'pull_request'
```

Comment:
No need to check if we're in a pull request for any of these; this workflow doesn't run in PRs.